### ISSUE TITLE
[Feature request] Italic text

### ISSUE THREAD

**Issue by KaKi87** [embark-framework/neo-blessed#46]
Hello,

As we have the ability to display bold, and underline text, I'd like to be able to display italic text.

Thanks !

**Comment by jhyland87** [comment]
I 1000% second this. I've spent a while trying to figure out why I can't get italic text to print in the terminal. I've tried the following:
- The `tput.sitm` function
- `{italic}Hello World{/italic}`
- `{sitm}Hello World{/sitm}`
- `\x1B[3mHello World\x1B[23m`
- All the different terminal types (_xterm-256color_, _xterm-color_, _xterm-new_, etc)
- Added another condition in the [Screen.prototype.codeAttr](https://github.com/chjj/blessed/blob/master/lib/widgets/screen.js#L1226) method (in screen.js) to check for the italic escape sequence (which is 3 to start and 23 to stop)
- Added another switch case in the [Screen.prototype.attrCode](https://github.com/chjj/blessed/blob/master/lib/widgets/screen.js#L1422) method (also in screen.js) to account for the italic escape sequence

And I know my terminal supports it because running `echo -e "\x1B[3mHello World\x1B[23m"` and `echo "$(tput sitm)Hello World$(tput ritm)"` both work just fine.
I can see an entry for `sitm` in the [lib/alias.js](https://github.com/chjj/blessed/blob/master/lib/alias.js#L154), but I can't get it to actually work.

The _really_ annoying part is that whenever I just print the object to the console and look at the `content` attribute of that object, I can see the correct italic sequences present. But they seem to get stripped before they get displayed, and I can't figure out how >_<

**Comment by jhyland87** [comment]
@KaKi87 & @embarkbot , do either of you know _how_ to fix the italic text issue? If you can point me in the right direction, I can try to fix/implement it and create a PR going

**Comment by KaKi87** [comment]
Nope, I actually even gave up on this, I am now only maintaining a CLI :/
